pkg/evpn/vrf.go

```diff
@@ -141,11 +141,31 @@ func (s *Server) CreateVrf(ctx context.Context, in *pb.CreateVrfRequest) (*pb.Vr
 		// configure FRR
 		data, err := utils.FrrZebraCmd(ctx, fmt.Sprintf(
 			`configure terminal
-			vrf %s-opi
-				vni 3%d
+			vrf %s
+				vni %d
 				exit-vrf
 			exit`, vrfName, *in.Vrf.Spec.Vni))
 		fmt.Printf("FrrZebraCmd: %v:%v", data, err)
+		data1, err1 := utils.FrrBgpCmd(ctx, fmt.Sprintf(
+			`configure terminal
+			router bgp 65000 vrf %s
```

Comment:
What gRPC config for BGP is expected from the user? this looks like we hardcode all the configs inside.

Reply:
The BGP config is out of scope right now. We care mostly about those minimal parameters that will come from the user that will make the EVPN work. So most of the BGP config is static right now.